pkg/scheduler/ugm/queue_tracker_test.go

```diff
@@ -155,3 +155,52 @@ func getQTResource(qt *QueueTracker) map[string]*resources.Resource {
 	usage := qt.getResourceUsageDAOInfo("")
 	return internalGetResource(usage, resources)
 }
+
+func TestQTCleanDeletedQueueTracker(t *testing.T) {
```

Comment:
`TestQTDeleteQueueTracker()`

Reply:
Good suggestion, addressed in latest PR.